### DISCUSSION TITLE
Can I disable type check on my file with the setting?

### DISCUSSION THREAD

**Discussion by VisionYi** [vuedx/languagetools#191] [Q&A] [answered]
Hi, vue-developer-experience's author

I want to add a setting like this below. Is it possible?

![image](https://user-images.githubusercontent.com/15682657/106392906-ee0c7880-642e-11eb-9b0c-2e826fa50e70.png)

I use this plugin on my Vue2 project, it is pretty useful!
But my project doesn't use typescript 😅. 
I want to disable these type of error messages like below
Have any idea or suggestion?

![image](https://user-images.githubusercontent.com/15682657/106393037-9d494f80-642f-11eb-8b95-47ea7b1c7a24.png)

Please, Thanks! 🥺

**Comment by znck** [comment, marked as answer]
No, it's out of scope for now.
